src/App.jsx

```diff
@@ -14,45 +16,115 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const ALPHABET_SORT = 'alphabetically';
+const LENGTH_SORT = 'length';
+const REVERSE = true;
+
+function getSortedGoods(goods, { sortType, reverse }) {
+  let preparedGoods = [...goods];
+
+  if (sortType) {
+    preparedGoods.sort((firstGood, secondGood) => {
+      switch (sortType) {
+        case ALPHABET_SORT:
+          return firstGood.localeCompare(secondGood);
+
+        case LENGTH_SORT:
+          return firstGood.length - secondGood.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverse === REVERSE) {
+    preparedGoods = preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+export const App = () => {
+  const [sortType, setsortType] = useState('');
+  const [reverse, setReverse] = useState(false);
+  const goodsCopy = getSortedGoods(goodsFromServer, { sortType, reverse });
```

Comment:
Will be more representive
```suggestion
  const sortedGoods = getSortedGoods(goodsFromServer, { sortType, reverse });
```